psi4/src/psi4/libfock/points.cc

```diff
@@ -250,7 +250,7 @@ void RKSFunctions::compute_orbitals(std::shared_ptr<BlockOPoints> block) {
     double** Ca2p = C_local_->pointer();
     for (int ml = 0; ml < nlocal; ml++) {
         int mg = function_map[ml];
-        C_DCOPY(na, Cap[ml], 1, Ca2p[mg], 1);
+        C_DCOPY(na, Cap[mg], 1, Ca2p[ml], 1);
```

Comment:
We might be able to track this back to the `memcpy` -> `DCOPY` move some time ago. Can you also fix UKS orbitals if needed?